api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -836,6 +836,11 @@ message Cluster {
   // CONNECT only if a custom filter indicates it is appropriate, the custom factories
   // can be registered and configured here.
   core.v3.TypedExtensionConfig upstream_config = 48;
+
+  // If track_request_response_sizes is true, then the :ref:`histograms
+  // <config_cluster_manager_cluster_stats_request_response_sizes>`  tracking header and body sizes
```

Comment:
let's align docs with what is implemented

Reply:
Can we merge #11766 with the current pr? Else aligning docs would need several redundant changes on both prs.